src/app/fdctl/run/tiles/fd_net.c

```diff
@@ -218,6 +234,29 @@ during_frag( void * _ctx,
   fd_memcpy( ctx->frame, src, sz ); // TODO: Change xsk_aio interface to eliminate this copy
 }
 
+static void
+send_arp_probe( void * _ctx,
+                uint   dst_ip_addr,
+                uint   ifindex ) {
+  fd_net_ctx_t * ctx     = (fd_net_ctx_t *)_ctx;
+  uchar *        arp_buf = ctx->arp_buf;
```

Comment:
why not just make this buf a stack local?

Reply:
I don't like to have too much on the stack

Reply:
Am I missing something, or it's 26 bytes? The stack is 8MiB. Having it in the context makes the purpose less clear I think.